install/install_openadapt.ps1

```diff
@@ -55,8 +59,7 @@ function RunAndCheck {
 function Cleanup {
     $exists = Test-Path -Path $setupdir
```

Comment:
Should not this line be changed to:

`$exists = Test-Path -Path $openAdaptPath`

?

Reply:
@KrishPatel13 
If we change that variable to `$openAdaptPath` as you are suggesting, which BTW is currently set to `"$env:USERPROFILE\OpenAdapt"` then it means that the installer will delete a non-existent `C:\User\<user>\OpenAdapt` folder which it hasn't used yet (which is fine), while trying to install dependencies i.e. the first 3 items in [this table](https://github.com/OpenAdaptAI/OpenAdapt/issues/447#issue-1829576911).

Reply:
👍